source/Stargate-Model/RESTfulRequestHandlerBuilder.class.st

```diff
@@ -53,11 +53,22 @@ RESTfulRequestHandlerBuilder >> build [
 	^  handlerFactory value
 ]
 
+{ #category : #configuring }
+RESTfulRequestHandlerBuilder >> createEntityHashingEncodedResource [
```

Comment:
Rename `createEntityHashingEncodedResource` to `createEntityTagHashingEncodedResource`